### PR TITLE
feat : beggin the Project

### PR DESCRIPTION
**Pull request by bhavanakoguru**
**Features:**

Static page: [JiraLink](https://basic-math123.atlassian.net/browse/MC-21)
       1.Home Component - [jira](JiraLink](https://basic-math123.atlassian.net/browse/MC-21)
       2.Welcome Component.[[jira]](JiraLink](https://basic-math123.atlassian.net/browse/MC-21)
       3.Nav Bar Components.[jira](JiraLink](https://basic-math123.atlassian.net/browse/MC-21)
       4.Profile Component.[jira](JiraLink](https://basic-math123.atlassian.net/browse/MC-21)
       5.Routing Component.[jira](JiraLink](https://basic-math123.atlassian.net/browse/MC-21)

I completed static functionalities for Home Components,Welcome Component,Nav Component,Nav Component,Profile Component,Rounting Component.

### CODE REVIEW

**Comment by vani-anigirappagari** [comment]
> **Features:**
> 
> Static page: [JiraLink](https://basic-math123.atlassian.net/browse/MC-21) 1.Home Component 2.Welcome Component. 3.NavBar Components. 4.Profile Component. 5.Routing Component.

PLease maintain proper description, JIra Id's.

File: Capstone_Project/frontEnd/package-lock.json
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/package-lock.json:1]: PLease remove package-lock json file

File: Capstone_Project/frontEnd/public/resources/profile.webp
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/public/resources/profile.webp:1]: PLease avoid pushing unsupported files

File: Capstone_Project/frontEnd/src/App.css
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/App.css:1]: PLease remove empty files.

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css [hunk 1]
```diff
@@ -0,0 +1,211 @@
+.totalPage
+{
+    display: flex;
+}
+/************/
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.css]: PLease remove unwanted code

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    margin-top: 10px;
+ }
+ .profile
+ {
+    width: 80px;
+    height: 80px;
+    margin-top: 350px;
+ }
+
+ /**************/
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.css]: Please remove unwanted code

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+.leftborder
+{
+   border: 1px solid black;
+   width: 350px;
+}
+.container2
+{
+   width: 100%;
+}
+.container2flex
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.css]: PLease maintain proper class name.

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    border-radius: 100px;
+    border: 1px solid black;
+    margin-right: 10px;
+    margin-top: 10px;
+}
+.c
+{
+    font-weight: 500;
+    width: 600px;
+    /* min-height: 10px;
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.css]: PLease remove unwanted code and use selectors, combinators

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx [hunk 5]
```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react"
+import './Homepage.css';
+import {Link} from "react-router-dom";
+
+function DisplayHome()
+{
+
+    const [start,setStart] = useState(false);
+
+    // function go()
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: Please remove unwanted code.

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+                .then(response => response.json())
+                .then(data => {
+                    console.log(data)
+                    setCourse(data)
+                })
+            }
+        }
+    }
+
+    const [units,setUnits] = useState([]);
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: PLease follow reactf file format.

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+                    setSubunits(subunitdata)
+                })
+        }
+    }
+    // function Click() {
+    //     setIsClicked(!isClicked);
+    //     if(isClicked){
+    //         fetch(`http://localhost:5050/Units`)
+    //             .then(response => response.json())
+    //             .then(subunitdata => {
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: PLease remove unwanted code.

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+                                    {
+                                        isClick ? 
+                                        course.map(({ course_Name }) => 
+                                        <div className="leftborder">
+                                                <h3 className='course' onClick={()=>clickMe()}>
+                                                    {course_Name}
+                                                </h3>
+                                        </div>)
+                                        :""
+                                        //  <><img src="/resources/miccymouse.svg" alt="mouse" className="mouse"/><h1 className="mouse1">WELCOME</h1></>
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: Please remove unwanted code.

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+                                                                <p className="content">{subUnit_Content}  </p> 
+                                                                
+                                                            </div>) 
+                                                        }
+                                                    </div> : ""
+                                                }
+                                            </div>
+                                        </div>
+                                        <div className="dropdown">
+                                            <img src="/resources/downarrow.png" alt="" className="b" />
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: please add alternate text for image tag.

File: Capstone_Project/frontEnd/src/components/Nav/navBar.jsx
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Nav/navBar.jsx:1]: PLease maintain proper folder names.

File: Capstone_Project/frontEnd/src/components/Profile/profile.css
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Profile/profile.css:1]: please remove unwanted files

File: Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.css [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    display: flex;
+}
+.welcome
+{
+    width: 1600px;
+}
+.nav{
+    width:80px;
+    height: 1100px;
+    background-color: rgb(237, 229, 229);
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.css:14]: PLease use selectors and combinators

File: Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.jsx [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+import { Link } from "react-router-dom";
+import "./WelcomePage.css"
+function Welcome()
+{
+    return(
+        <div className="totalPage">
+        <div className="nav">
+               <img src="/resources/jtd.jpeg" alt="LogoJTD" className="logo" />
+               <Link to="/home"><img src="/resources/booklive.jpeg" alt="livebook" className="LogoLive livebook" /></Link>
+                {/* <img src="/resources/booklogo.jpeg" alt="livebook" className="livebook" /> */}
```
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.jsx]: Please remove unwanted code

File: Capstone_Project/frontEnd/src/index.css
Comment (vani-anigirappagari) [inline Capstone_Project/frontEnd/src/index.css:1]: PLease remove unwanted file

File: Capstone_Project/frontEnd/package-lock.json
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/package-lock.json:1]: Resolved

File: Capstone_Project/frontEnd/src/App.css
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/App.css:1]: removed

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css (hunk 1 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.css]: Resolved

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css (hunk 2 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.css]: resolved

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css (hunk 3 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.css]: resolved

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx (hunk 5 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: Resolved

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx (hunk 7 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: Resolved

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx (hunk 9 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: Resolved

File: Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.css (hunk 10 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.css:14]: Okay .I will Use

File: Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.jsx (hunk 11 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Welcome/WelcomePage.jsx]: Resolved

File: Capstone_Project/frontEnd/src/index.css
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/index.css:1]: Resolved

File: Capstone_Project/frontEnd/src/components/Home/Homepage.jsx (hunk 6 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.jsx]: Resolved

File: Capstone_Project/frontEnd/src/components/Home/Homepage.css (hunk 4 above)
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Home/Homepage.css]: Resolved

File: Capstone_Project/frontEnd/src/components/Nav/navBar.jsx
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Nav/navBar.jsx:1]: Resolved

File: Capstone_Project/frontEnd/src/components/Profile/profile.css
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/src/components/Profile/profile.css:1]: This is Wanted File. I will Write Code Today

File: Capstone_Project/frontEnd/public/resources/profile.webp
Comment (bhavanakoguru) [reply Capstone_Project/frontEnd/public/resources/profile.webp:1]: This image is supporting in VS code platform I will check and Push Again

**Review by kirankumar-medikurthy-jtd — APPROVED** [review]

**Pull request merged**